File: aisile-parent/aisile-sellergoods-interface/src/main/java/com/aisile/sellergoods/service/BrandService.java
package com.aisile.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.aisile.pojo.TbBrand;
import com.aisile.pojo.entity.PageResult;

public interface BrandService {
	//查询全部
     public List<TbBrand> findAll();
     //分页
     public PageResult findPage(int pageNum,int pageSize);
     //增加
     public void add(TbBrand brand);
     //修改
     public void update(TbBrand brand);
     //回显
     public TbBrand findOne(Long id);
     //删除
     public void delete(Long [] ids);
     //分页+模糊
 	public PageResult findSearch(TbBrand brand, int pageNum,int pageSize);
 	//模板里的下拉列表
 	public List<Map> selectOptionList();
 	
}

File: aisile-parent/aisile-manager-web/src/main/webapp/js/service/brandService.js
//封装所有连接后端的代码,需要$http
	 app.service('brandService',function($http){
		//查询全部
		 this.findAll = function(){
			 return $http.get('../brand/findAll.do');
		 }
		this.findPage = function(page,rows){
			return $http.get('../brand/findPage.do?page='+page+'&rows='+rows);
		}
		//带参数分页查询
		this.search = function(page,rows,searchEntity){
			return $http.post('../brand/search.do?page='+page+"&rows="+rows,searchEntity);
		}
		//修改
		this.update = function(entity ){
			return $http.post('../brand/update.do',entity);
		}
		//添加
		this.add = function(entity){
			return $http.post('../brand/add.do',entity );
		}
		//删除
		this.dele = function(selectIds){
			return $http.get('../brand/delete.do?ids='+selectIds);
		}
		//根据id查询对象
		this.findOne = function(id){
			return $http.get('../brand/findOne.do?id='+id);
		}
		//模板里的下拉列表
		this.selectOptionList = function(){
			return $http.get('../brand/selectOptionList.do');
		}
	 })